src/app/components/SearchForm/SearchForm.jsx

```diff
@@ -32,27 +32,25 @@ class SearchForm extends React.Component {
   submitSearchRequest(event) {
     event.preventDefault();
     if (!this.state.searchField) {
-      this.setState({ searchField: 'q' });
+      this.setState({ searchField: 'keyword' });
     }
     if (!this.state.searchQuery) {
       throw new Error('Please enter a term or terms to search');
     }
 
     const terms = this.state.searchQuery.trim().replace(/\s+/g, ' ');
 
-    if (!this.state.searchField || this.state.searchField === 'q') {
-      this.props.searchGet(terms);
-    } else {
-      this.props.searchPost(terms, this.state.searchField);
-    }
-    this.context.router.push(`/search?q=${terms}`);
+    this.props.searchPost(terms, this.state.searchField)
+      .then(() => {
+        this.context.router.push(`/search?q=${terms}`);
```

Comment:
Would it make sense to show the searchField here instead of just "q"?

Reply:
Am exposing the field as a second parameter in the next PR for the Title, Author and to be added Subject fields since we'll need to pass those along. More URL params to come when adding sort, count per page, page, etc.